doc/content/sqa/mastodon_cci.md

```diff
@@ -0,0 +1,27 @@
+!template load file=app_cci.md.template app=MASTODON category=mastodon
+
+!template! item key=general-communication
+The best way to contact the MASTODON team is through the [mastodon-users google group](https://groups.google.com/forum/#!forum/mastodon-users).
+
+To join the group, GMail users can just click the "Join group" button.
+Non-GMail users must send an email to:
```

Comment:
This if fine for now, but you should considering switching to GitHub Discussions.